mediathread/templates/dashboard/base_course_dashboard.html

```diff
@@ -2,6 +2,19 @@
 
 {% block title %}&mdash; Manage Course {% endblock %}
 
+{% block messages %}
+    {% if messages %}
+        {% for message in messages %}
+        <div class="alert alert-warning alert-dismissible fade show" role="alert">
```

Comment:
This looks good. The only change -- I'd say that we should use "alert-info" rather than warning here as the messages are mostly confirming an action.